docs/zh_CN/Compressor/Pruner.md

```diff
@@ -14,6 +14,12 @@ NNI Compressor 中的 Pruner
     * [APoZ Rank Pruner](#activationapozrankfilterpruner)
     * [Activation Mean Rank Pruner](#activationmeanrankfilterpruner)
 
+* [具有梯度等级的 Filter Pruners](#gradientfilterpruners)
```

Comment:
We do not need to update Chinese doc, translation will be done in bunch for each release.

Reply:
yes. @squirrelsc this is the translated one.